src/client/modules/main/addons/mute/Mute.js

```diff
@@ -62,7 +69,29 @@ class Mute {
 		if (muteChar == v) return Promise.resolve(false);
 
 		return this.muteChars.set({ [charId]: muteChar || undefined }).then(() => {
-			this._saveMuteChars();
+			this._saveMuteChars('.chars', this.muteChars);
```

Comment:
Nice deduplication of code by adding `_saveMute`  (and  `_loadMute`). But here is a bug. Should be:
 `this._saveMute(`
 instead of
 `this_saveMuteChars(`

Reply:
Yep! Good catch. Missed it while reading through myself.